tests/foreman/cli/test_remoteexecution.py

```diff
@@ -197,3 +242,22 @@ def test_positive_run_scheduled_job_template(self):
 
         @Assert: Verify the job was successfully ran after the designated time
         """
+        system_current_time = ssh.command('date +"%b %d %Y %I:%M%p"').stdout[0]
+        current_time_object = datetime.strptime(
+            system_current_time, '%b %d %Y %I:%M%p')
+        plan_time = (current_time_object + timedelta(seconds=30)).strftime(
+            "%Y-%m-%d %H:%M")
+        invocation_command = make_job_invocation({
+            'job-template': 'Run Command - SSH Default',
+            'inputs': 'command="ls"',
+            'start-at': plan_time,
+            'search-query': "name ~ {0}".format(self.client.hostname),
+        })
+        max_attempts = 5
```

Comment:
seems a little bit redundant variable
`for _ in range(5):` probably should suffice our needs